extensions/amp-bind/amp-bind.md

```diff
@@ -98,7 +98,7 @@ For AMP components, only select attributes are bindable:
 | Component | Attributes |
 | --- | --- |
 | amp-carousel | slide |
-| amp-img | src, srcset, alt |
+| amp-img | src, srcset, alt, height, width |
```

Comment:
Then should [this](https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/0.1/bind-validator.js#L399) include the other three supported tags?